tests/FINDOLOGIC/Export/Tests/XmlSerializationTest.php

```diff
@@ -37,6 +38,15 @@ public static function setUpBeforeClass()
         self::$schema = file_get_contents(self::SCHEMA_URL);
     }
 
+    public function tearDown()
+    {
+        try {
```

Comment:
Is a try catch-block actually needed as there is no need to check if the file was deleted?

Reply:
In case the file to delete doesn't exist, an exception would be thrown. This guards against it for all the tests that don't create export files.

✅